### PR TITLE
Remove deprecated HTML

### PR DESCRIPTION
**Pull request by DivaVocals**
* Remove deprecated language="javascript usage
* Update doctype so that it's consistent across all admin files
* Remove outdated comment markup around script tags

### CODE REVIEW

File: includes/modules/pages/account_newsletters/jscript_main.php [hunk 1]
```diff
@@ -9,8 +9,8 @@
...
  */
 ?>
-<script><!--
+<script>
 function checkBox(object) {
   document.account_newsletter.elements[object].checked = !document.account_newsletter.elements[object].checked;
 }
-//--></script>
\ No newline at end of file
+</script>
```
Comment (drbyte) [inline includes/modules/pages/account_newsletters/jscript_main.php]: Please add a new blank line at the end of this file.

**Comment by drbyte** [comment]
Thanks for this PR. :)

1. Most of the jscript_xxxx.js and jscript_xxxx.php files are missing the blank line at the end of the file. 
A vague guess is there's maybe 10+ files like that.  May as well do that as part of this PR.

2. In the admin files you changed the doctype from html4 to html5 on several files. What sort of browser testing did you do on that?

**Comment by DivaVocals** [comment]
> Most of the jscript_xxxx.js and jscript_xxxx.php files are missing the blank line at the end of the file. A vague guess is there's maybe 10+ files like that. May as well do that as part of this PR.

Will take a look later today or tomorrow and make this fix. I think my "find and replace" was a tad bit aggressive.. :laughing: 

>In the admin files you changed the doctype from html4 to html5 on several files. What sort of browser testing did you do on that?

Actually this is a change I've sort of routinely done to all of my own Zen Cart installs, but admittedly not across the WHOLE admin as I've done here.. After looking at some of the v1.6 files, I "assumed" (perhaps incorrectly) that this was a general direction being moved towards with the changes being made to the admin. I'll back those changes out though.. :smile:

**Comment by texdc** [comment]
@drbyte What is the concern with `<DOCTYPE html>`?  What browsers would be incompatible with that?
@DivaVocals nice job! :+1:

**Comment by DivaVocals** [comment]
> Most of the jscript_xxxx.js and jscript_xxxx.php files are missing the blank line at the end of the file. 
So I need coffee because I didn't spot this myself.. After rolling things back I found that many of these files may not have had a blank line in them to begin with..

**Comment by drbyte** [comment]
@DivaVocals 
> After rolling things back I found that many of these files may not have had a blank line in them to begin with.. 

Correct. That wasn't your fault. :)

> They do now as a part of this PR

Thanks!

**Comment by drbyte** [comment]
> I too would like to understand why a move to HTML5 or even XHTML in the admin would be discouraged/unwanted.. OR were ya just trying to make me think here?

I was just asking what testing you've done on it. I'm not opposed to a proper move to HTML5 ... but there's much more to it than merely a doctype change. Granted, the new (albeit smallish) template stubs in the admin are a step in that direction, but even without removing all the (yucky!) tables, there are still additional considerations needed plus lots of cross browser testing (mainly looking out for small quirks, as "big" issues are probably a lesser risk).

So, ya, just asking the status, not objecting.

**Comment by drbyte** [comment]
Argh, I see that when you reverted the doctype changes, you undid even more than what were originally there.

**Comment by DivaVocals** [comment]
Right.. There were already SOME doctype changes in place.. I simply updated everything else.. So.. let me start over.. Gonna submit JUST the <script> tag changes and that will be that..

**Comment by DivaVocals** [comment]
>I was just asking what testing you've done on it. I'm not opposed to a proper move to HTML5 ... but there's much more to it than merely a doctype change. Granted, the new (albeit smallish) template stubs in the admin are a step in that direction, but even without removing all the (yucky!) tables, there are still additional considerations needed plus lots of cross browser testing (mainly looking out for small quirks, as "big" issues are probably a lesser risk).

>So, ya, just asking the status, not objecting.

Gotcha.. Just checking.. :smile_cat: 

Yeah removing the tables is a HUGE undertaking, and perhaps I was a little TOO enthusiastic in my pull request by making the doctype change without examining the full file.. So I'll go at it one (or two) files at a time.. :smile:  For example I've done some "spot changes" with regards to removing tables from some admin files and changing the doctype of those files.. If it's okay I will start submitting pull requests with some of the simpler low hanging fruit versions of some of these changes I routinely have done for my clients.. (like removing the tables from the search forms at the top of the orders.php page..)

**Comment by drbyte** [comment]
Just FYI, when doing these changes, please keep admin and catalog in separate PRs.  If there's no direct dependency between them it's easier if they're kept separate.

(For example, changing doctype stuff on catalog side is really a whole separate project unto itself, as it's a totally separate templating system ... etc)

**Comment by DivaVocals** [comment]
Got it.. But let me ask this, if it's a pull request with functionality changes that impact the catalog side which requires admin changes to support, it IS okay if these are part of the same pull request right??

**Comment by drbyte** [comment]
Yes

**Comment by DivaVocals** [comment]
Groovy!!! Since I am not a TRUE code master like the rest of you rock stars here, I may generally stick with contributions for things that are related to HTML kinds of stuff..

Thanks for your guidance.. This has been a whole lot of fun (NOT! :laughing:) stumbling and stuttering my way through my first pull request..

**Comment by drbyte** [comment]
No worries. Happy to help you get started ... everyone has to start somewhere!

Thanks for taking initiative and making the contribution!

**Comment by DivaVocals** [comment]
Oh I am happy to help!! Excited that I get to be a part of "the club" :laughing: My issue up until now has been having TIME to do this.. Working full time and maintaining my clients sites keeps me BUSY.. I've got time now - since I rolled off my full-time consultant engagement (which is a nice way to say I'm temporarily unemployed and looking for work :laughing:)

**Pull request closed without merging**